tron/core/job.py

```diff
@@ -336,20 +332,15 @@ def run_job(self, job_run, run_queued=False):
         # If the Job has been disabled after this run was scheduled, then cancel
         # the JobRun and do not schedule another
         if not self.job.enabled:
-            log.info("%s cancelled because job has been disabled." % job_run)
+            log.info(f"Cancelled {job_run} because job has been disabled.")
             return job_run.cancel()
 
         # If the JobRun was cancelled we won't run it.  A JobRun may be
         # cancelled if the job was disabled, or manually by a user. It's
         # also possible this job was run (or is running) manually by a user.
         # Alternatively, if run_queued is True, this job_run is already queued.
         if not run_queued and not job_run.is_scheduled:
-            log.info(
-                "%s in state %s already out of scheduled state." % (
-                    job_run,
-                    job_run.state,
-                )
-            )
+            log.info(f"Scheduling {job_run} in state {job_run.state}")
```

Comment:
I think this might be misleading, since we're actually scheduling a different job run in the next line? 
How about, `f"{job_run} in state {job_run.state} is not scheduled, scheduling a new run instead of running"` or something?